templates/display.html

```diff
@@ -22,17 +22,16 @@ <h4>There {{posts.count|pluralize:"is,are"}} {{posts.count}} Syllab{{posts.count
     <form method="POST">
       {% csrf_token %}
       <input name="pk" value="{{post.pk}}" type="hidden"/>
-      <a href="https://{{AWS_S3_CUSTOM_DOMAIN}}/{{post.syllabus}}" class="btn btn-success" target="_blank">{{ post.dept }} {{ post.number }} taught by {{post.prof}}
+      <a href="https://{{AWS_S3_CUSTOM_DOMAIN}}/{{post.syllabus}}" class="btn btn-success" target="_blank">{{ post.dept }} {{ post.number }} taught by {{post.prof}} </a>
       {% if post in user.profile.saved.all %}
-      <button type="submit" name="unsave" value="Unsave" class="btn btn-sm">
+      <button type="submit" name="unsave" value="Unsave" title="Unsave" alt="Remove from saved" class="btn btn-link">
```

Comment:
The alt needs to be on the `img` tag, sorry that I wasn't clear about that

Reply:
nah im dumb lmao